server/JsonStreamingServer/suppliers/JsonStreamingServer.Suppliers.FileStream/JsonStreamingServer.Suppliers.FileStream.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CsvHelper" Version="30.0.1" />
```

Comment:
IMO no need for this package since CSV is just a text file you can use file stream with a stream reader and read lines one by one and then map those lines to a C# object.